persist.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -u
+set -e
+dir=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+
+. ${dir}/utils
+PROJECT=${PROJECT:-$(get_project_name)}
+ACTION=backup
```

Comment:
Maybe use a different variable name since the user might pass "ACTION" when running ``.omero/docker`` and get confused